helpers/handlebars.js

```diff
@@ -212,6 +212,16 @@ hbs.registerHelper("md_partial", function md_partial(partial, opts) {
 	return final;
 });
 
+hbs.registerHelper("get_police_encounter", function log(police) {
+	const startingStr = "As you arrive at the hotel, you see ";
+
+	if (police.officers > 1) {
+		return `${startingStr} ${police.officers} polices arrive behind you. They have ${police.total_hp} HP.`;
+	}
+
+	return `${startingStr} the police arrive behind you. The police has ${police.total_hp} HP.`;
```

Comment:
Should say "...have X HP" instead of "...has"

Reply:
If I change to `officer`, should I be using ".....have X HP" or "...has?"

Reply:
Yeah, I guess if you change it to "officer", has will be fine.